service/controller/clusterapi/v27/cloudconfig/master_template.go

```diff
@@ -93,6 +95,7 @@ type RandomKeyTmplSet struct {
 }
 
 type MasterExtension struct {
+	awsConfigSpec g8sv1alpha1.AWSConfigSpec
```

Comment:
That base extension and master extension and what not is all super confusing and ugly somehow but I am just trying to make it work for now.